### PR TITLE
Set istio replication channel tunneling to false

### PR DESCRIPTION
**Pull request by jshum2479**
Set tunneling to false for istio replication channel per weblogic security recommendations.

### CODE REVIEW

**Review by lennyphan — APPROVED** [review]
LGTM

File: operator/src/main/resources/scripts/model_wdt_mii_filter.py [hunk 1]
```diff
@@ -531,12 +531,12 @@ def customizeIstioReplicationChannel(server, name, listen_address):
...
                  listen_port=istio_repl_listen_port, protocol='t3', http_enabled="true",
                  bind_to_localhost="true", use_fast_serialization='true',
-                 tunneling_enabled='true')
+                 tunneling_enabled='false')
   else:
     _writeIstioNAP(name='istiorepl', server=server, listen_address=listen_address,
                    listen_port=istio_repl_listen_port, protocol='t3', http_enabled="true",
                    bind_to_localhost="false", use_fast_serialization='true',
-                   tunneling_enabled='true')
+                   tunneling_enabled='false')
```
Comment (robertpatrick) [inline operator/src/main/resources/scripts/model_wdt_mii_filter.py:539]: This file needs the copyright date updated

File: operator/src/test/python/test_wdt_mii_filter.py [hunk 2]
```diff
@@ -364,6 +364,7 @@ def test_customizeIstioReplicationChannel_version_1_10(self):
       istiorepl_nap = model['topology']['ServerTemplate']['cluster-1-template']['NetworkAccessPoint']['istiorepl']
       self.assertIsNotNone(istiorepl_nap, "Expected a NAP named \'istiorepl\' to be defined")
 
+      self.assertEqual(istiorepl_nap['TunnelingEnabled'], 'false')
```
Comment (robertpatrick) [inline operator/src/test/python/test_wdt_mii_filter.py:367]: This file seems to be missing the copyright header altogether...

**Review by robertpatrick — CHANGES_REQUESTED** [review]
Please update the copyrights

File: operator/src/main/resources/scripts/model_wdt_mii_filter.py (hunk 1 above)
Comment (jshum2479) [reply operator/src/main/resources/scripts/model_wdt_mii_filter.py:539]: done

File: operator/src/test/python/test_wdt_mii_filter.py (hunk 2 above)
Comment (jshum2479) [reply operator/src/test/python/test_wdt_mii_filter.py:370]: done

**Review by robertpatrick — APPROVED** [review]

**Pull request merged**